### DISCUSSION TITLE
linkerd check fails after successful pre check and installation

### DISCUSSION THREAD

**Discussion by aleksandarjnec** [linkerd/linkerd2#11026] [Help]
I am getting this error after installing the latest stable version in HA mode in my AWS EKS.
Any idea how to solve it?

> × control plane CustomResourceDefinitions exist
>     CRD httproutes.policy.linkerd.io is missing version v1beta3
>     see https://linkerd.io/2/checks/#l5d-existence-crd for hints

**Comment by adleong** [comment]
@aleksandarjnec was this an upgrade from a previous version of Linkerd?  Did you run `linkerd upgrade --crds` to upgrade the Linkerd crds?
